src/js/popup.js

```diff
@@ -255,6 +258,17 @@ const Logic = {
       }
       return identity;
     });
+    const containerOrderStorage = await browser.storage.local.get([CONTAINER_ORDER_STORAGE_KEY]);
```

Comment:
Can you make this part of the `promise.all` above and then merge it into `_identities` in the `map()`?

Reply:
I suspect we should treat this code as if there is always an order rather than make it a little more complex if there is an optional sort order. I also think to make it deterministic we should always add a sort order when a new container is added.

Reply:
done